kong/clustering/init.lua

```diff
@@ -22,6 +24,37 @@ local ngx_WARN = ngx.WARN
 local _log_prefix = "[clustering] "
 
 
+-- Sends "clustering", "push_config" to all workers in the same node, including self
```

Comment:
Could we put those config sync related codes into a separate file?

Reply:
Will be done on following refactoring PR.